src/components/popover/react/Popover.tsx

```diff
@@ -119,216 +106,291 @@ const CLASSNAME: string = getRootClassName(COMPONENT_NAME);
+    /**
+     * Handle mouse entering the popover.
+     */
+    const handleMouseEnter = (): void => {
+        setIsMouseEntered(true);
     };
 
-    return (
-        <Manager>
-            <Reference>
-                {({ ref }: ReferenceChildrenProps): ReactElement => (
-                    <div
-                        ref={(elm: HTMLDivElement | null): void => {
-                            anchorRef = elm;
-                            ref(elm);
-                        }}
-                        style={{ width: 'fit-content', height: 'fit-content' }}
-                        onMouseEnter={onMouseEnteredReference}
-                        onMouseLeave={onMouseLeftReference}
-                    >
-                        {anchorElement}
-                    </div>
-                )}
-            </Reference>
-            {(unwrap(showPopper) || (unwrap(useTooltipMode) && autoShowPopper)) && (
-                <Popper placement={popperPlacement as PopperPlacement} modifiers={modifiers}>
-                    {({ ref, style, arrowProps, ...others }: PopperChildrenProps): ReactElement => {
-                        const computedOffsets: Position = popperPlacement
-                            ? computeOffsets(others.placement, popperPlacement as PopperPlacement, popperOffset)
-                            : {};
-                        const computedSizes: Size = computeSize(
-                            fillHeight,
-                            fillwidth,
-                            style.transform,
-                            matchAnchorWidth,
-                        );
-                        return (
-                            <div
-                                ref={ref}
-                                style={{ ...style, ...computedOffsets, ...computedSizes }}
-                                className={classNames(
-                                    handleBasicClasses({
-                                        elevation: elevation && elevation > 5 ? 5 : elevation,
-                                        prefix: useTooltipMode ? 'lumx-tooltip' : CLASSNAME,
-                                    }),
-                                )}
-                                x-placement={popperPlacement}
-                            >
-                                {popperElement}
-                                {useTooltipMode && (
-                                    <div
-                                        ref={arrowProps.ref}
-                                        style={arrowProps.style}
-                                        className="lumx-tooltip__arrow"
-                                    />
-                                )}
-                            </div>
-                        );
-                    }}
-                </Popper>
+    /**
+     * Handle mouse leaving the popover.
+     */
+    const handleMouseLeave = (): void => {
+        setIsMouseEntered(false);
+    };
+
+    useEffect(() => {
```

Comment:
Why not put all this useEffect in the useComputePosition ?
If you do so, you can get rid of the "value" fake state

Reply:
Also, we could try to externalize the state to make the component fully controlled.
In the demo we could initialize the Popover with a position (x, y, width, height) instead of the anchorRef and call the useCompustePosition externally.